build-scripts/compile_profiles.py

```diff
@@ -54,7 +54,7 @@ def get_env_yaml(build_config_yaml, product_yaml):
     if build_config_yaml is None or product_yaml is None:
         return None
 
-    env_yaml = ssg.yaml.open_environment(build_config_yaml, product_yaml)
+    env_yaml = ssg.environment.open_environment(build_config_yaml, product_yaml)
```

Comment:
I notice that in other files where you did this change, you also altered the import statement. Here you did not. Nothing seems to break during build, but I rather ask.

Reply:
The script didn't have an `import ` for the `yaml` module. So I just followed the same format and replaced `ssg.yaml` with `ssg.environment`.

Reply:
I think this happens because `ssg.build_yaml` already imports `ssg.{yaml->environment}` and so you don't strictly need to duplicate the import.